Taxi/Taxi/Presenter/AddTaxiParty/AddTaxiParty.swift

```diff
@@ -165,6 +195,22 @@ extension AddTaxiParty {
         }
     }
 
+    private func midDaySelector(_ day: MorningAfternoon) -> some View {
+        Button {
+            if selectedMidDay != day {
+                selectedMidDay = day
+                startHour = nil
+            }
+        }
```

Comment:
이전 리뷰에서 말한 버그는 여기에 `startMinute = nil`을 추가하면 해결할 수 있습니다!
```suggestion
    private func midDaySelector(_ day: MorningAfternoon) -> some View {
        Button {
            if selectedMidDay != day {
                selectedMidDay = day
                startHour = nil
                startMinute = nil
            }
        }
```

Reply:
반영 완료하였습니다~~!!